modules/7_ibmcloud/dns.tf

```diff
@@ -30,7 +30,7 @@ resource "ibm_dns_record" "bastion" {
   count              = var.bastion_count
   data               = var.bastion_ip[count.index]
   domain_id          = data.ibm_dns_domain.domain.id
-  host               = "${var.cluster_id}-bastion-${count.index}"
+  host               = "${var.name_prefix}-bastion-${count.index}"
```

Comment:
This is not needed if we are not changing the hostname and sticking with only resource name change for adding zone.

Reply:
Closing this comment as we decided to have zone name in hostname for the nodes.